### DISCUSSION TITLE
exec /usr/local/sbin/bootstrap: no such file or directory

### DISCUSSION THREAD

**Discussion by evilscytheman** [lloesche/valheim-server-docker#557] [Q&A]
Portainer gives me this log. What went wrong? What can i do

**Comment by wcdanh2** [comment]
can you post the details of your portainer configuration?

**Comment by evilscytheman** [comment]
Sorry, windows noob here :)
I don't know exactly what you mean.
I installled portainer via:
`sudo docker volume create portainer_data
sudo docker run -d -p 8000:8000 -p 9443:9443 --name portainer \
    --restart=always \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v portainer_data:/data \
    portainer/portainer-ce:latest`

after that I installed the valheim server via the instruction within the documentation "[Basic Docker Usage](https://hub.docker.com/r/lloesche/valheim-server#basic-docker-usage)"
